backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interest/ModularContractAllocations.java

```diff
@@ -145,7 +145,7 @@ private Instant getInterimDate()
 			return cachedInterimContractDate;
 		}
 
-		cachedInterimContractDate = interimContractEntry.getTransactionDate().toInstant(orgDAO::getTimeZone);
+		cachedInterimContractDate = modularContractLogEntry.getTransactionDate().toInstant(orgDAO::getTimeZone);
```

Comment:
Does this still make sense? This is now the modular contract date, not the interim contract date (which did not make sense either, because the contract date != interim invoice date).
Maybe we should think about moving to use the interim date parameter from the process?

Reply:
done